osbrain/tests/test_nameserver.py

```diff
@@ -124,8 +123,8 @@ def test_nameserver_proxy_shutdown_lazy_agents(delay, timeout):
     Shutdown a name server proxy with agents that wait some time before
     shutting down.
 
-    The name server should always wait for all agents to shut down when no
-    timeout is set.
+    The name server should always succeed. If the agents do not shut down
```

Comment:
Missing "shutdown" word? So it would be: "The name server shutdown should always succeed."?

Reply:
@Flood1993 You think it is not clear with the paragraph just above? (Where we talk about the name server shutdown).

Just in case you did not read it... If you did but think it is not clear, I will add it. :blush:

Reply:
I hadn't seen it, but still I think it makes more sense to add it there, for clarity 👍